.github/workflows/chart-e2e.yaml

```diff
@@ -127,7 +129,7 @@ jobs:
           fi
           if ! helm install --create-namespace --namespace $NAMESPACE --wait \
               --timeout "$ROLLOUT_TIMEOUT_SECONDS" \
-              --set autodependency.enabled=true ${EXTRAPARAM} \
+              --set autodependency.enabled=true \ #${EXTRAPARAM} \
```

Comment:
I this this is syntax error for calling helm. You don't need to change this line